HarryPotter/Screens/Character/ViewController/CharacterViewController.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  CharacterViewController.swift
+//  HarryPotter
+//
+//  Created by Adrianna Mikulska on 10/09/2022.
+//
+
+import UIKit
+
+class CharacterViewController: UICollectionViewController {
+    
+    //MARK: - Properties
+    private let characterCellId = "characterCellId"
+    private let  characterViewModel: CharacterViewModelProtocol
+    
```

Comment:
Try to keep it nice & clean (one empty space between `// MARK: -` and the properties